.github/workflows/release.yaml

```diff
@@ -20,6 +20,8 @@ jobs:
       - name: Create Release
         id: create_release
         uses: actions/create-release@v1
+        # Otherwise we can't rerun this to build new binaries
+        continue-on-error: true
```

Comment:
This can be useful if the workflow succeeds but we want to rebuild the binaries currently this will fail.